pkg/cmd/kafka/acl/list/list.go

```diff
@@ -141,6 +149,40 @@ func runList(opts *options) (err error) {
 		req = req.Principal(principalQuery)
 	}
 
+	var selectedResourceTypeCount int
+	var resourceType string
+	var resourceName string
+
+	if opts.topic != "" {
+		selectedResourceTypeCount++
+		resourceType = aclutil.ResourceTypeTOPIC
+		resourceName = opts.topic
+	}
+
+	if opts.group != "" {
+		selectedResourceTypeCount++
+		resourceType = aclutil.ResourceTypeGROUP
+		resourceName = opts.group
+	}
+
+	if opts.cluster {
+		selectedResourceTypeCount++
+		resourceType = aclutil.ResourceTypeCLUSTER
+		resourceName = aclutil.KafkaCluster
+	}
+
+	if selectedResourceTypeCount > 1 {
+		return opts.localizer.MustLocalizeError("kafka.acl.common.error.oneResourceTypeAllowed", flagutil.ResourceTypeFlagEntries...)
```

Comment:
This error message specifies transactional-id as an option, but it is not valid here.

```shell
❯ ./rhoas kafka acl list --group "enda" --topic "enda"
❌ Provide only one resource type from "--cluster", "--topic", "--group", or "--transactional-id". Run the command in verbose mode using the -v flag to see more information
```